### DISCUSSION TITLE
Background Job not working

### DISCUSSION THREAD

**Discussion by jrahma** [shinyorg/shiny#1222] [Q&A]
I am trying to run a background job in the attached sample but it's not working.

The job should insert a record in a remote Microsoft SQL Server but that i snot happening.

Kindly help..

Thanks,
Jassim
[ContactsBackground.zip](https://github.com/shinyorg/shiny/files/11435449/ContactsBackground.zip)

**Comment by aritchie** [comment]
This is not where issues are filed or how.  Please be respectful of the code of conduct. 

https://github.com/shinyorg/shiny/issues/new/choose
